spec/models/landing_page_spec.rb

```diff
@@ -42,4 +45,11 @@
       expect(described_class.new(content_item).blocks.first.type).to eq(expected_type)
     end
   end
+
+  describe "#menus" do
```

Comment:
This test needs to be updated to use `navigation_groups`, which I think is a much better name than "menus". 🎉

Reply:
I... don't understand what's going on here. On my local machine, with these commits, this method is called `#navigation_groups`, and the comment below also says that.

Reply:
Ah, sneaky fix commit got merged in later on where I couldn't spot it easily. Fixed.